frontend/app/src/bounties/BountyDescription.tsx

```diff
@@ -166,7 +166,7 @@ const BountyDescription = (props: BountiesDescriptionProps) => {
               {props.title?.length > 80 ? '...' : ''}
             </EuiText>
           </div>
-          {descriptionImage && (
+          {/* {descriptionImage && (                  commented out reason-> #1059
```

Comment:
why are we removing this? @ecurrencyhodler we need this if there is an image in git that we want to show

Reply:
@kevkevinpal I think that's what is causing the bounty on the homepage to display that image. We should keep it @gouravmpk . The fix is to prevent the image in the description from being displayed in the bounties home page. It should still be displayed in the bounty itself.